weaviate-services/agents/transformation-agent-get-started.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>In the docs I used something like this:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">for operation in response:
    print(agent.get_status(workflow_id=operation.workflow_id))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/weaviate/recipes/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>